.travis.yml

```diff
@@ -13,3 +13,12 @@ cache:
   - node_modules
 after_success:
   - npm run coverage
+script:
+    - npm test
+    - if [[ ALPINE = 1 ]]; then docker build . -t pidusage && npm run alpine; fi
```

Comment:
Nice.
Maybe we should use the build matrix to test on all the OS.
This would be fantastic.

Reply:
Yes my though as well I just want to see if this goes well first then we will absolutely do this.

Reply:
A nice list of distros:
https://github.com/retrohacker/getos/blob/master/os.json